landscaper/pkg/controlplane/configuration.md

```diff
@@ -400,7 +392,7 @@ gardenerControllerManager:
 
 ### Component configuration for the Gardener Admission Controller
 
-The component configuration of the Gardener Admission Controller is optional, as default values will be provided.
+The component configuration of the Gardener Admission Controller is optional and enabled by default.
```

Comment:
What it means a component configuration to be enabled?

```suggestion
The component configuration of the Gardener Admission Controller is optional, as default values will be provided.
```

Reply:
Thanks, bad formulation.